package.json

```diff
@@ -108,6 +113,7 @@
     "firefox-extension-deploy": "1.1.1",
     "flow-bin": "0.82.0",
     "html-loader": "0.5.5",
+    "husky": "^1.1.4",
```

Comment:
This should be a fixed version.

Reply:
I'll make that change. Does it also make sense to add a line to CONTRIBUTING.md, at the top-level or as a sub-section under the 'Adding New Files' section, saying something like "When adding a new dependency or devDependency, make sure to fix the version in package.json"?

Reply:
We should make a linter that runs in CI and pre push :grin:

Reply:
And also document it to contributing.md